### DISCUSSION TITLE
Autoplay with no rec. songs

### DISCUSSION THREAD

**Discussion by Liteaus** [EvieePy/Wavelink#256] [Q&A] [answered]
I want to make a queue system for my Discord bot, but I don't want it to automatically play suggested songs. I tried using autoplay but it plays suggested songs when the track ends. I just want it to play the songs was queued. Could you please help me ?, thank you.

i use wavelink 2.6.5

**Comment by EvieePy** [comment, marked as answer]
This is only a feature in version 3 which is now the latest.
